docs/csharp/whats-new/csharp-11.md

```diff
@@ -91,12 +91,12 @@ These types aren't directly represented in metadata. They include annotations th
 
 There are several language features that enable generic math support:
 
-- static virtual members in interfaces
+- static abstract members in interfaces
 - checked user defined operators
 - relaxed right-shift requirements
```

Comment:
Maybe `relaxed shift operators` is a better term? This features relaxes a requirement around the right hand side of a `left shift`, `signed (arithmetic) right shift`, or `unsigned (logical) right shift`. Previously it was required that the signature was `T operator <<(T left, int right)`, that is the right hand side must be `int`. Now it can be any type.